src/main/webapp/js/const.es6

```diff
@@ -16,4 +42,98 @@ const Const = {
 
     },
 
+    // Used in instructorFeedback.js
+    InstructorFeedback: {
```

Comment:
I'd prefer if the constants are broken up by category instead of their usages. Something like:
```js
ParamsNames: {
    ...
},

StatusMessages: {
    ...
},
```
i.e. much like `Const.java`.

Also, remove the `// Used in ...`s because they are very fragile as a comment.